packages/fxa-payments-server/src/components/PlanDetails/index.tsx

```diff
@@ -73,16 +74,14 @@ export const PlanDetails = ({
 
   return (
     <div
-      className={`plan-details-component ${className}`}
+      className={`plan-details-component text-base my-8 mx-0 p-0 ${className}`}
```

Comment:
Let me know your thoughts on this, but I think we can remove `text-base`. It's affecting the overall `font-size` of the children elements making all text appear larger and more compressed (especially the list elements). We can add individual `font-size`s to all children elements. Unless there's another reason you think we should keep it (other than it was there before)?

Reply:
Aligned! I think we can take out `font-normal` too, as that is also a default.